hx_lti_initializer/static/vendors/development/video.dev.js

```diff
@@ -6309,6 +6320,10 @@ vjs.TextTrackMenuItem = vjs.MenuItem.extend({
 vjs.TextTrackMenuItem.prototype.onClick = function(){
   vjs.MenuItem.prototype.onClick.call(this);
   this.player_.showTextTrack(this.track.id_, this.track.kind());
+  if (typeof(jQuery.publish) === "function") {
```

Comment:
Ahh ok, yeah that gets pretty tricky then if it has changed before you even got it. Good to make a note anyway.

[Able Player](https://ableplayer.github.io/ableplayer/) looks interesting. I definitely understand the need to have something more accessible these days.